iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Substation.java

```diff
@@ -25,9 +26,9 @@
     Network getNetwork();
 
     /**
-     * Get the country to which the substation belongs.
+     * Get an optional containing the country to which the substation belongs if it is defined, Optional.empty() if not.
      */
-    Country getCountry();
+    Optional<Country> getCountry();
```

Comment:
Don’t forget to flag your PR as it breaks the API. Could you create the « breaking change » label?

Reply:
Done